index.html

```diff
@@ -115,11 +115,11 @@ <h1>Tonic</h1>
 
     <div class="card reverse proj_2">
       <div class="img proj_#2">
-        <img src="images/Project_2.svg" alt="project image" />
+        <img src="images/Project_2.svg" alt="Image for project 2 small screen"/>
       </div>
 
       <div class="card_Desktop">
-        <img src="images/desk_2.svg" alt="desk_image">
+        <img src="images/desk_2.svg" alt="image for project 2 desktop version">
```

Comment:
- [ ] An unexpected horizontal scroll appears when I resize the website, kindly fix it.

![image](https://user-images.githubusercontent.com/60164239/131633151-1e7807bd-f850-407d-985c-c11c6248c0a0.png)

Best practices according to:

![image](https://user-images.githubusercontent.com/60164239/131633431-28c0f561-f784-4dfd-a10e-033c393e8286.png)

Reply:
- [ ] [OPTIONAL] You can consider centering the content (as you can see in the image below for your desktop version), it will be nice to implement 😉 

![image](https://user-images.githubusercontent.com/60164239/131694069-dd76d55d-3b14-4f11-a7e8-fc88d5b44970.png)